basics/a11y.md

```diff
@@ -2,67 +2,94 @@
 
 ## Background info
 
-[The A11y Project](https://a11yproject.com/about)
+Accessibility (shortened as a11y) is a primary considering in our design and engineering processes. By foregrounding
+a11y, we intend to make our web properties more equitable and usable for everyone.
+
+The Libraries' Accessibility Team (LAT), established in 2023, has adopted [Inclusive Design](https://www.nngroup.com/articles/inclusive-design)
+as one of its guiding principles. Inclusive Design is similar to [Universal Design](https://universaldesign.ie/about-universal-design),
+in that both explore the idea of meeting as many people’s a11y needs as possible. Inclusive Design focuses more on individual needs, and acknowledges that multiple design approaches may be required to meet this goal.
+
+We consider Inclusive Design to be an inherently aspirational goal, as more can always be done. This is also what makes
+it a useful framing device for a11y work. The human experience is incredibly diverse, and a single design is not likely to meet every person's a11y needs. We can always strive to do better.
 
 ## Goals
 
 - The site is usable by everyone.
-- Intentional access restrictions like MIT only are okay.
+- The site meets as many people's a11y needs as possible.
 - Anyone that is authorized to use a site can actually use it.
+  - Intentional access restrictions (like 'MIT users only') are okay.
+- The site is compliant with [WCAG 2.1 AA](https://www.w3.org/TR/2023/REC-WCAG21-20230921/) guidelines.
```

Comment:
I considered making this WCAG 2.1 AAA, which is what the DSP specifies. It feels somewhat aspirational, but we could include that here. (This is a list of goals, not requirements, so aspirational could make sense..)

Reply:
I agree AA makes sense here for now.